### ISSUE TITLE
Sabres package

### ISSUE THREAD

**Issue by Wytamma** [LilWinx/Sabres#10]
Would it be possible to get Sabres into a conda package or on pip? That would make it much easier to use in our pipelines. I’d be happy to submit a PR to do this!

**Comment by Wytamma** [comment]
Hey @LilWinx, I've made Sabres into a pip installable package here -> https://github.com/Wytamma/Sabres/tree/package

You can install it from Github with the following command:

```
pip install git+https://github.com/Wytamma/Sabres@package
```

This install sabres as a cli so you can run `sabres --help` from the command line to get
```
usage: sabres [-h] [--full] [--lineage LINEAGE] --vcall {ivar,varscan,medaka}
              --outdir OUTDIR --input INPUT [--version]

Sabres

optional arguments:
  -h, --help            show this help message and exit
  --full, -f            Use Full Database
  --lineage LINEAGE, -l LINEAGE
                        Add Lineage Information
  --vcall {ivar,varscan,medaka}, -v {ivar,varscan,medaka}
                        Specify variant caller software used
  --outdir OUTDIR, -o OUTDIR
                        Output directory to write to
  --input INPUT, -i INPUT
                        Input directory or file
  --version             get SABRes version
```

I'll make a PR for this shortly :)

**Comment by LilWinx** [comment]
Hi @Wytamma,

Thank you for raising this issue, we are in the works of pushing v1.1 in the next day or two, which is why I haven't done conda/pip.

**Comment by Wytamma** [comment]
No worries @LilWinx! I'll hold off on a PR until you have v1.1 then rebase to bring in any changes. I started adding automated [tests](https://github.com/Wytamma/Sabres/blob/package/tests/test_cli.py) to that package branch also so hopefully that can save you some time.

**Comment by LilWinx** [comment]
Merged PR#15
